lib/Driver/ToolChains.cpp

```diff
@@ -1044,6 +1044,24 @@ addLinkSanitizerLibArgsForDarwin(const ArgList &Args,
                      /*AddRPath*/ true, TC);
 }
 
+static void
+addLinkFuzzerLibArgsForDarwin(const ArgList &Args,
+                       ArgStringList &Arguments,
+                       const ToolChain &TC) {
+
+  // libFuzzer requires C++.
+  Arguments.push_back("-lc++");
+
+  // Link libfuzzer.
+  SmallString<128> Dir;
+  getRuntimeLibraryPath(Dir, Args, TC);
+  llvm::sys::path::remove_filename(Dir);
+  llvm::sys::path::append(Dir, "llvm", "libLLVMFuzzer.a");
```

Comment:
You are right. Actually for testing I am just using compiler built out of the `build` directory: there, everything works. This is already a large productivity gain over manually compiling and linking libFuzzer.